index.html

```diff
@@ -80,29 +80,72 @@ <h3>Location</h3>
 </p>
 
 
+<h2>Processing Model</h2>
+
+<p>
+Many procedural specifications deal with data passing from script into some browser function where it is processed, and then some output is returned back into script. For both specifications and browser implementations, it is important to understand how different languages participate in this.
+</p>
+
+<p>
+A typical browser implementation interacts with script as follows:
+</p>
+
+<ol>
+  <li>A <em>script engine</em> executes a line of JavaScript code.</li>
+  <li>A <em>bindings layer</em> translates this into a call on a C++ object, translating the input types from script to C++.</li>
```

Comment:
I'd say "an internal object (generally C++ or Rust)" and similarly change the later reference to "C++" to say "internal" or similar.